client/blocks/reader-post-card/featured-asset.jsx

```diff
@@ -2,37 +2,65 @@
  * External Dependencies
  */
 import React from 'react';
-import { head, filter, get } from 'lodash';
+import { find, endsWith, some, } from 'lodash';
+import url from 'url';
 
 /**
  * Internal Dependencies
  */
 import FeaturedVideo from './featured-video';
 import FeaturedImage from './featured-image';
 
+function isCandidateForFeature( media ) {
+	if ( ! media ) {
+		return false;
+	}
+
+	if ( media.mediaType === 'image' ) {
+		const image = media;
+
+		const imageIsTallEnough = 350 <= image.width || 350 <= image.naturalWidth;
+		const imageIsWideEnough = 85 <= image.height || 85 <= image.naturalHeight;
+
+		return imageIsTallEnough && imageIsWideEnough;
+	} else if ( media.mediaType === 'video' ) {
+		// we need to have a thumbnail and know how to autoplay it
+		return media.thumbnailUrl && media.autoplayIframe;
+	}
+
+	return true;
+}
+
+// jetpack lies about thumbnails so we need to make sure its actually an image. See: thumbIsLikelyImage in pick-canonical-image rule
+function isUrlLikelyAnImage( uri ) {
```

Comment:
yeah, should be fine. I try to avoid requires that cross into other things utils though...  if > 1 thing needs a util, it should just be in lib/thing.

so i'd make a new `lib/is-url-likely-an-image` and use that from both places